templates/takeovers/index.html

```diff
@@ -138,5 +138,6 @@
 {% include "takeovers/_snapd-takeover.html" %}
 <p>22 May 2020</p>
 {% include "takeovers/_cio-guide-to-multipass-fr.html" %}
-
+<p>%script:name=28 May 2020</p>
```

Comment:
Looks like this was a mistake: `%script:name=`

Reply:
I have been having some random paste issues lately... I wonder what is happing.. Atom, Regolith or me!